internal/database/actions.go

```diff
@@ -256,14 +255,14 @@ func CreateResult(sess sqlbuilder.Database, homePath string, testGroupName strin
 }
 
 // CreateNetwork will create a new network in the network table
-func CreateNetwork(sess sqlbuilder.Database, location *model.LocationInfo) (*Network, error) {
+func CreateNetwork(sess sqlbuilder.Database, loc util.LocationProvider) (*Network, error) {
```

Comment:
This is an interface rather than `*session.Session` to simplify mocking and testing.

Reply:
Despite this comment, I had a tough time understanding that this is indeed a reference to `probe-engine:session.Session`. But that was my poor knowledge of how things work in Go. It does make sense now. Thanks for walking me through it.